src/bullsAndCows.js

```diff
@@ -26,7 +26,33 @@
  * @return {object} - like {bulls: 0, cows: 0}
  */
 function bullsAndCows(generatedNumber, enteredNumber) {
-  // write code here
+  const bullObject = {
+    'bulls': 0,
+    'cows': 0,
+  };
+
+  if (typeof (enteredNumber) === 'undefined') {
+    return undefined;
+  }
+
+  if (enteredNumber.length !== 4 || enteredNumber.length === undefined) {
```

Comment:
нет смысла во второй проверке, оставшуюся проверку можно совместить с той что выше
```suggestion
  if (enteredNumber.length !== 4) {
```